fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIORoot.scala

```diff
@@ -35,10 +35,12 @@ sealed trait BIORootInstancesLowPriority1 extends BIORootInstancesLowPriority2 {
 
 sealed trait BIORootInstancesLowPriority2 extends BIORootInstancesLowPriority3 {
   @inline implicit final def BIOConvertFromBIOProfunctor[FR[-_, +_, +_]](implicit BIOProfunctor: NotPredefined.Of[BIOProfunctor[FR]]): BIOFunctor3[FR] with S3 = S3(BIOProfunctor.InnerF)
+  @inline implicit final def BIOConvertFromBIOParallel[FR[-_, +_, +_]](implicit BIOParallel: NotPredefined.Of[BIOParallel3[FR]]): BIOMonad3[FR] with S3 = S3(BIOParallel.InnerF)
```

Comment:
```suggestion
  @inline implicit final def BIOConvertFromBIOParallel[FR[-_, +_, +_]](implicit BIOParallel: NotPredefined.Of[BIOParallel3[FR]]): BIOMonad3[FR] with S5 = S3(BIOParallel.InnerF)
```
And better to move `BIOProfunctor` conversion to a lower priority level, because otherwise they will conflict if there's both constraints at the same time `F[-_, +_, +_]: BIOProfunctor: BIOParallel` - we wantConvertFromBIOParallel to have a higher priority because it will return more specific BIOMonad instead of BIOFunctor

Reply:
Have moved it to priority 3